articles/api/management/v2/get-access-tokens-for-spas.md

```diff
@@ -38,83 +38,7 @@ With a Management API Token issued for a SPA, you can access the following scope
 | `create:current_user_device_credentials` | [POST /api/v2/device-credentials](/api/management/v2#!/Device_Credentials/post_device_credentials) |
 | `delete:current_user_device_credentials` | [DELETE /api/v2/device-credentials/{id}](/api/management/v2#!/Device_Credentials/delete_device_credentials_by_id) |
 
-::: note
 The above scopes (and endpoints) are [rate limited](/policies/rate-limits#access-tokens-for-spas).
-:::
-
-## Using a Management API Token to call the Management API from a SPA
```

Comment:
I'm not sure **why** to remove this section if we have still documented the use of mgmt api access token in SPAs in the earlier section. 

If we removing this section, we should remove SPA reference and only recommend using mgmt api access_token on the backend.

Reply:
Added back in but removed code snippets.